src/modules/apps/testCommand.ts

```diff
@@ -0,0 +1,161 @@
+import retry from 'async-retry'
```

Comment:
These dependencies have to be on package.json, otherwise we will not be able to test this plugin without being on toolbelt